Sources/PortalSwift/Components/PortalWebView.swift

```diff
@@ -44,7 +44,7 @@ public class PortalWebView: UIViewController, WKNavigationDelegate, WKScriptMess
   private var onError: (Result<Any>) -> Void
   private var onPageStart: (() -> Void)?
   private var onPageComplete: (() -> Void)?
-
+    
```

Comment:
Nit: Added whitespace here, is your linter working by chance? I'm happy to add a commit with the `swiftformat` changes if you want 🙂

Reply:
Ah let me get that configured correctly and re-run it. Thanks!

Reply:
Done!